src/MoBi.Presentation/Presenter/CreateSimulationConfigurationPresenter.cs

```diff
@@ -50,17 +49,19 @@ public class CreateSimulationConfigurationPresenter : WizardPresenter<ICreateSim
          AllowQuickFinish = false;
       }
 
-      public SimulationConfiguration CreateBasedOn(IMoBiSimulation moBiSimulation, bool allowNaming = true)
+      public SimulationConfiguration CreateBasedOn(IMoBiSimulation moBiSimulation, bool isNew = true)
       {
-         edit(moBiSimulation, allowNaming);
+         edit(moBiSimulation, isNew);
          UpdateControls();
          _view.Display();
          if (_view.Canceled)
          {
             return null;
          }
 
-         var simulationConfiguration = _simulationConfigurationFactory.Create();
+         // *Configuring* a simulation with new modules and building blocks should not replace the simulation settings
+         var simulationConfiguration = _simulationConfigurationFactory.Create(settings: isNew ? null : moBiSimulation.Settings);
```

Comment:
here. you are passing the settings from another simulation. Feels wrong

Reply:
Maybe this should be a clone of the settings then. The idea is to create a simulation configuration but use the settings, or a copy of the settings from an existing simulation.

In this specific use-case, they go back to the same simulation because configure creates a new simulationConfiguration with new building blocks, but obviously that's not enforced from within this method.

I think if a clone is created of the settings, then it should work even if they're used in a different simulation.

Reply:
Yes
BEcause otherwise, editing one simulation would change the other. I do not know what's the original simulation is but a clone would make it obvious